src/Microsoft.SqlTools.Hosting/Utility/CommandOptions.cs

```diff
@@ -46,6 +46,9 @@ public CommandOptions(string[] args, string serviceName)
                             case "-application-name":
                                 ApplicationName = args[++i];
                                 break;
+                            case "-application-path":
```

Comment:
nit: I'd suggest naming this something like `data-path` to match what VS Code uses